src/utils/qmln.ts

```
@@ -0,0 +1,631 @@
+    }
+
+    toString() {
+        return Array.from(this.packet).map(b => b.toString(16).padStart(2, '0')).join(' ').toUpperCase();
+    }
+}
+
+export class Function {
+    static functionCode = 0x00;
+    messageBody: Uint8Array;
+
+    constructor() { }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)